src/map.c

```diff
@@ -53,29 +54,28 @@ SEXP call_loop(SEXP env, SEXP call, int n, SEXPTYPE type, int force_args,
     SEXP res = PROTECT(R_forceAndCall(call, force_args, env));
 
     if (type != VECSXP && Rf_length(res) != 1) {
-      stop_bad_element_length(res, i + 1, 1, "Result", NULL, false);
+      Rf_errorcall(R_NilValue, "Result must be length 1, not %i.", Rf_length(res));
```

Comment:
`r_abort()`?

Reply:
These deliberately use `Rf_errorcall` because all the necessary context is already included in the wrapper.